frontend-react/src/components/DataDashboard/ReportDetails/ReportDetailsSummary.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { Grid } from "@trussworks/react-uswds";
+import moment from "moment";
```

Comment:
I remember discussing this with @jpandersen87 previously, but aren't we trying to move away from `moment.js` since it's completely unmaintained now? You can see on their homepage: https://momentjs.com/

Might not to be handled here, but we should discuss if we want to be using `moment` going forwards, and if not, a ticket to replace it across our app.

Reply:
Yes, good catch. Will update.